app/design/adminhtml/default/default/template/catalog/product/helper/gallery.phtml

```diff
@@ -13,101 +13,136 @@
+                                type="radio" <?php if ($this->getElement()->getAttributeReadonly($typeId)) : ?> disabled="disabled" <?php endif ?>
+                                name="<?php echo $type['field'] ?>"
+                                onclick="<?php echo $this->getJsObjectName(); ?>.setProductImages('no_selection')"
+                                value="no_selection"/></td>
                 <?php endforeach ?>
-                <td class="cell-disable"><input type="hidden" />&nbsp;</td>
-                <td class="cell-remove last"><input type="hidden" />&nbsp;</td>
-        </tr>
-    </tbody>
-<?php if (!$_block->getElement()->getReadonly()):?>
-    <tfoot>
-        <tr>
-            <td colspan="100" class="last" style="padding:8px">
-                <?php echo Mage::helper('catalog')->__('Maximum width and height dimension for upload image is %s.', Mage::getStoreConfig(Mage_Catalog_Helper_Image::XML_NODE_PRODUCT_MAX_DIMENSION)); ?>
-                <?php echo $_block->getUploaderHtml() ?>
-            </td>
-        </tr>
-    </tfoot>
-<?php endif ?>
-</table>
-</div>
+                <td class="cell-disable"><input type="hidden"/>&nbsp;</td>
+                <td class="cell-remove last"><input type="hidden"/>&nbsp;</td>
+            </tr>
+            </tbody>
+            <?php if (!$this->getElement()->getReadonly()): ?>
+                <tfoot>
+                <tr>
+                    <td colspan="100" class="last" style="padding:8px">
+                        <?php echo Mage::helper('catalog')->__('Maximum width and height dimension for upload image is %s.', Mage::getStoreConfig(Mage_Catalog_Helper_Image::XML_NODE_PRODUCT_MAX_DIMENSION)); ?>
+                        <?php echo $this->getUploaderHtml() ?>
+                    </td>
+                </tr>
+                </tfoot>
+            <?php endif ?>
+        </table>
+    </div>
 </div>
-<input type="hidden" id="<?php echo $_block->getHtmlId() ?>_save" name="<?php echo $_block->getElement()->getName() ?>[images]" value="<?php echo $_block->escapeHtml($_block->getImagesJson()) ?>" />
-<input type="hidden" id="<?php echo $_block->getHtmlId() ?>_save_image" name="<?php echo $_block->getElement()->getName() ?>[values]" value="<?php echo $_block->escapeHtml($_block->getImagesValuesJson()) ?>" />
+<input type="hidden" id="<?php echo $this->getHtmlId() ?>_save"
+       name="<?php echo $this->getElement()->getName() ?>[images]"
+       value="<?php echo $this->escapeHtml($this->getImagesJson()) ?>"/>
+<input type="hidden" id="<?php echo $this->getHtmlId() ?>_save_image"
+       name="<?php echo $this->getElement()->getName() ?>[values]"
+       value="<?php echo $this->escapeHtml($this->getImagesValuesJson()) ?>"/>
 <script type="text/javascript">
 //<![CDATA[
 var <?php echo $_block->getJsObjectName(); ?> = new Product.Gallery('<?php echo $_block->getHtmlId() ?>', <?php echo $_block->getImageTypesJson() ?>);
```

Comment:
i can't see gallery cause in this template $_block is undefined
Warning: Undefined variable $_block in app/design/adminhtml/default/default/template/catalog/product/helper/gallery.phtml on line 148

Reply:
need to replace $_block with $this

Reply:
fixed here https://github.com/fballiano/openmage/pull/21/commits/a1553fe4cddb0afed75d153848784633a963f81a